posthog/queries/trends/sql.py

```diff
@@ -52,7 +52,7 @@
 # :TODO: Fix this!
```

Comment:
I wonder if this change counts to remove this comment 🤣

Reply:
Nope. Problem is lines 66 and 69 below as we group every timestamp separately.

I'm not confident enough I understand the semantics of this query enough to refactor it to something like `toStartOfDay(timestamp)` :/